app/src/org/commcare/preferences/CCPrefValues.java

```diff
@@ -0,0 +1,26 @@
+package org.commcare.preferences;
+
+import android.util.DisplayMetrics;
+
+/**
+ * Created by amstone326 on 11/14/17.
+ */
+public class CCPrefValues {
```

Comment:
What do you think about naming these `PrefValues` and `ServerUrls`

Reply:
sure